### ISSUE TITLE
Allow features on a map to be tab navigable 

### ISSUE THREAD

**Issue by ahmadayubi** [Maps4HTML/Web-Map-Custom-Element#268]
It was suggested that the features should be tab navigable by users if they are within the map viewport.
